task7.css

```diff
@@ -0,0 +1,26 @@
+body {
+    padding: 10px;
+    margin: 0;
+}
+
+header {
+    background-color: #008b66;
+    color: #ffffff;
+    padding: 10px;
+    z-index: -1;
```

Comment:
if element has position static z-index is not apply
http://htmlbook.ru/css/position